feature-libs/cart/quick-order/components/quick-order/form/quick-order-form.component.html

```diff
@@ -1,4 +1,8 @@
-<form [formGroup]="form" class="quick-order-form-container row">
+<form
+  (ngSubmit)="search($event)"
```

Comment:
Did Bill ask for this button? and does it go with your wireframes? I haven't seen the wireframes though

However, if we are committing to this button, do we need line 4, specifically, `    <input
      (keydown.enter)="search($event)"`. Do we even need `keydown.enter` on the input when we have a button?  

------
I think we should stick to our 'search style' input that we have, where we have an input and the search icon at end, such as the searchbox input and the storefinder input

![Screen Shot 2021-07-26 at 12 18 02 AM](https://user-images.githubusercontent.com/29099466/126932542-e9477e85-424a-4cda-8584-ef7a19f6df34.png)
![Screen Shot 2021-07-26 at 12 18 07 AM](https://user-images.githubusercontent.com/29099466/126932544-86e558ac-0243-4168-9ca8-9337b466660d.png)

Reply:
Had a call with Manu and get solution for this.

Reply:
https://github.com/SAP/spartacus/pull/13245#discussion_r678589452